ethportal-api/Cargo.toml

```diff
@@ -54,6 +54,7 @@ validator = { version = "0.13.0", features = ["derive"] }
 env_logger = "0.9.0"
 quickcheck = "1.0.3"
 rstest = "0.16.0"
+test-case = "3.3.1"
```

Comment:
We use [rstest](https://docs.rs/rstest/latest/rstest/index.html) for test fixtures but I like `test-case` so no blocker on my side for adding this crate.

Reply:
This is somewhat of a *nitpick*, but I do think there's a strong argument to remain consistent throughout the codebase w/ regards to what libraries we're using. The one exception I can think of is when we use `surf` instead of `ureq` for http requests, b/c `ureq` has poor async support. It does seem to me like `rstest` and `test-case` are doing essentially the same function...

Reply:
> This is somewhat of a _nitpick_, but I do think there's a strong argument to remain consistent throughout the codebase w/ regards to what libraries we're using. The one exception I can think of is when we use `surf` instead of `ureq` for http requests, b/c `ureq` has poor async support. It does seem to me like `rstest` and `test-case` are doing essentially the same function...

I agree but it is a development dependency, we don't require this on a release so i'm not strongly against using both libraries.

Reply:
I'm with @njgheorghita on this one. It is just dev dependency but there are advantages in having consistent codebase.
I just wasn't aware of rstest. Done